### PR TITLE
Fix spelling of access=customer_s_ and add to the restricted list

### PR DESCRIPTION
**Pull request by TheMarex**
# Issue

Fixes the following routing bug in Tiergarten Berlin:

broken

## Tasklist
 - [x] add regression / cucumber cases (see docs/testing.md)
 - [ ] review
 - [ ] adjust for comments

### CODE REVIEW

**Comment by emiltin** [comment]
why is customers in both the blacklist and list of restricted tags?
should this be added to bike and foot profiles?

**Comment by TheMarex** [comment]
@emiltin the blacklist entry makes sure highways that use these tags need to be on `restricted_highway_whitelist` or will get removed. We added this specifically for access restricted non-car roads, that we would have deemed routable (e.g. like in this case a `foodpath`).

File: profiles/car.lua [hunk 1]
```diff
@@ -84,7 +84,8 @@ local profile = {
   restricted_access_tag_list = Set {
     'private',
     'delivery',
-    'destination'
+    'destination',
+    'customers',
```
Comment (daniel-j-h) [inline profiles/car.lua]: Related: https://github.com/Project-OSRM/osrm-backend/issues/3745

We should probably just add the fee tag there, too.

Comment (TheMarex) [inline profiles/car.lua]: Yup we should, but would do this in a separate PR. 👍
